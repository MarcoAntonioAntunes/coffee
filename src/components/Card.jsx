import React from 'react';

const Card = ({ giver, receiver }) => {
    return (
        <div className="card">
            <h3>{giver} </h3>
            <p className="link">should give a cup of coffee to</p>
            <h3>{receiver} </h3>
        </div>
    );
};

export default Card;