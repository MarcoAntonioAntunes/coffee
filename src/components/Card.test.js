import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Card from './Card';


it('should match the Card  component to snapshot', () => {
    const component = renderer.create(
        <Card />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

it('should render the Card component', () => {
    configure({ adapter: new Adapter() });
    const component = shallow(
        <Card />
    );
    const componentChildDiv = component.children().getElements()[1];
    expect(componentChildDiv.props.className).toEqual('link');
});