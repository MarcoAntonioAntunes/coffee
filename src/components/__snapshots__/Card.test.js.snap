// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the Card  component to snapshot 1`] = `
<div
  className="card"
>
  <h3>
     
  </h3>
  <p
    className="link"
  >
    should give a cup of coffee to
  </p>
  <h3>
     
  </h3>
</div>
`;
